### PR TITLE
Create autotest to ensure parameters are always documented

### PR DESCRIPTION
**Pull request by peterbarker**
The following parameters are not documented and will cause this PR to fail to pass autotest ATM.

ArduPlane:
```
ICE_IDLE_DB not in XML
AFS_WP_GPS_LOSS not in XML
SOAR_POLAR_B not in XML
SOAR_Q2 not in XML
SOAR_Q1 not in XML
SOAR_POLAR_CD0 not in XML
SOAR_R not in XML
```

ArduCopter: None

Helicopter: 
```
H_SW_H3_SV3_POS not in XML
```

ArduSub: None

APMrover2:
```
WRC2_RATE_FLTD not in XML
WRC2_RATE_FLTE not in XML
WRC2_RATE_FLTT not in XML
WRC_RATE_FLTE not in XML
WRC_RATE_FLTD not in XML
WRC_RATE_FLTT not in XML
```

Tracker: too many to list

### CODE REVIEW

File: libraries/AP_Scripting/AP_Scripting.cpp [hunk 1]
```diff
@@ -71,6 +71,11 @@ const AP_Param::GroupInfo AP_Scripting::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("HEAP_SIZE", 3, AP_Scripting, _script_heap_size, SCRIPTING_HEAP_SIZE),
 
+    // @Param: DEBUG_LVL
+    // @DisplayName: Enable Scripting Debug
+    // @Description: Enable scripting debug.
+    // @User: Advanced
```
Comment (amilcarlucas) [inline libraries/AP_Scripting/AP_Scripting.cpp]: A range and increment whould be helpfull.

Comment (peterbarker) [reply libraries/AP_Scripting/AP_Scripting.cpp]: Oops.  That patch went in in a different PR.  PRs welcome to fix it in master :-)

If you really think every single parameter deserves those attributes modifications to the code being merged in this PR could relatively easily ensure it.

**Comment by peterbarker** [comment]
Closes #13561

**Comment by rmackay9** [comment]
Thanks for the fixes to the wheel encoder parameters!

**Pull request merged**